### PR TITLE
Implement boskos metrics

### PR DESCRIPTION
**Pull request by krzyzacy**
also fixed some mal-formatted double quotes in the doc

Those are very basic metrics, and will fill up more stuff in follow up PRs.

/assign @fejta

### CODE REVIEW

File: boskos/ranch/ranch.go [hunk 1]
```diff
@@ -301,3 +301,35 @@ func (r *Ranch) SaveState() {
...
+	for _, res := range r.Resources {
+		if res.Type != rtype {
+			continue
+		}
+
+		if _, ok := metric.Current[res.State]; !ok {
+			metric.Current[res.State] = 0
+		}
+
+		if _, ok := metric.Current[res.State]; !ok {
```
Comment (cjwagner) [inline boskos/ranch/ranch.go]: s/Current[res.State]/Owners[res.Owner]

Comment (krzyzacy) [inline boskos/ranch/ranch.go]: awwww good catch, fixed and added test coverage for the case.

**Comment by krzyzacy** [comment]
/retest

https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/test-infra/3226/pull-test-infra-bazel/4380/ did not understand that :-(

**Pull request merged**